test/d-b-n_smoke/arrays.html

```diff
@@ -22,79 +22,99 @@
+                        <h4>
+                            <span>{{model.name}}</span> people:</h4>
+                        <ul class="people">
+                            <template is="dom-repeat" items="{{model.people}}">
+                                <li>
+                                    <span>{{item.first}}</span>
+                                </li>
                             </template>
-                        </li>
-                        <li class="length">{{model.people_properties.length}}</li>
-                        <li class="update">{{model.people_properties.prop}}</li>
-                        <template is="dom-if" if="{{model.people_properties.newProp}}" restamp>
-                            <li class="add">Add</li>
-                        </template>
-                        <template is="dom-if" if="{{model.people_properties.deletedProp}}" restamp>
-                            <li class="delete">Deleted? - <span>{{model.people_properties.deletedProp}}</span></li>
-                        </template>
-                    </ul>
-                    <ul class="push">
-                        <template is="dom-repeat" items="{{model.people_push}}">
-                            <li>
-                                <span>{{item.first}}</span>
-                            </li>
-                        </template>
-                    </ul>
-                    <ul class="pop">
-                        <template is="dom-repeat" items="{{model.people_pop}}">
-                            <li>
-                                <span>{{item.first}}</span>
-                            </li>
-                        </template>
-                    </ul>
-                    <ul class="splice">
-                        <template is="dom-repeat" items="{{model.people_splice}}">
-                            <li>
-                                <span>{{item.first}}</span>
-                            </li>
-                        </template>
-                    </ul>
-                    <ul class="splice_n_change">
-                        <template is="dom-repeat" items="{{model.people_splice_n_change}}">
-                            <li>{{item.first}}</li>
-                        </template>
-                    </ul>
-                    <ul class="primitive_splice_n_change">
-                        <template is="dom-repeat" items="{{model.primitive_splice_n_change}}">
-                            <li>{{item}}</li>
-                        </template>
-                    </ul>
-                    <ul class="people_nested">
-                        <template is="dom-repeat" items="{{model.people_nested}}">
-                            <li>
-                                <template is="dom-repeat" items="{{item}}">
+                        </ul>
+                        <ul class="primitive_people">
+                            <template is="dom-repeat" items="{{model.primitive_people}}">
```

Comment:
Wouldn't Polymer replace it with `<ul><dom-repeat><template...`? If so, I would replace it here as well, to have less magic under cover. If not, I would keep it as I like `<tempalte is="` use cases :)

Reply:
Wow, I wonder how this passed.  This is probably a mistake.